### PR TITLE
[Content/Fix] Adding missing documentation content

### PR DESCRIPTION
**Pull request by ronijaakkola**
## Description
Adding missing content to the new documentation site. Also fixing some smaller issues reported in the accessibility audit.
- Added Pagination docs
- Added indeterminate Checkbox docs
- Added dense Koros docs
- Added Visual assets/Icons docs and the icon list
- Removed redundant `<br>` tags from many pages
- Fixed DateInput examples to have unique ids
- Replaced What's new `<strong>` tags with headings
- Fixed Logo documentation alt-texts
- Fixed an issue where Linkbox examples were out of the screen on smaller viewports
- Fixed splitted `<li>` structures on the Designers' contribution page
- Added a `lang` property to Finnish wave motif names
- Changed "Jump to content" link to move focus to the main heading of the content area

## How Has This Been Tested?
Tested by running the documentation site locally.

### CODE REVIEW

**Review by minevala — COMMENTED** [review]
Huge set of fixes 👍 
A few suggestions and couple of things to fix

File: new-site/src/docs/foundation/visual-assets/icons/list.mdx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+import * as HDS from "hds-react";
+
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## List of all icons
+
+### Arrows and operators
+Icon                        | Core              | React             | Common Common usage
```
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply new-site/src/docs/foundation/visual-assets/icons/list.mdx:15]: Could it be possible to control that via the table title syntax we are using?

File: new-site/src/docs/components/pagination/code.mdx [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+      </li>
+    </ul>
+    <button type="button" class="hds-button hds-button--supplementary hds-button--theme-black hds-pagination__button-next">
+      <span class="hds-button__label">Next</span>
+      <span aria-hidden="true" class="hds-icon hds-icon--angle-right"></span>
+    </button>
+  </nav>
+</div>
+
+<div class="hds-pagination-container">
```
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply new-site/src/docs/components/pagination/code.mdx:136]: - [ ] Removed redundant code from the pagination truncation example

File: new-site/src/docs/components/pagination/tabs.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+# Pagination
+
+<StatusLabel type="info">Stable</StatusLabel>
+<StatusLabel type="success" style={{ marginLeft: 'var(--spacing-xs)' }}>
+  Accessible
+</StatusLabel>
+
+<LeadParagraph>
+  Toggle button allows the user to switch between two distinct states such as "On" and "Off".
```
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply new-site/src/docs/components/pagination/tabs.mdx]: - [ ] Fixed the Pagination documentation page LeadParagraph

File: new-site/src/docs/components/checkbox/index.mdx [hunk 4]
```diff
@@ -4,6 +4,7 @@ title: 'Checkbox'
 nav_title: 'Checkbox'
 ---
 
+import { useReducer } from 'react';
 import { Checkbox, SelectionGroup } from 'hds-react';
```
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply new-site/src/docs/components/checkbox/index.mdx]: - [ ] Add a missing Fieldset import to the Checkbox documentation page

File: new-site/src/docs/foundation/visual-assets/icons/index.mdx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+### Illustrative icons and pictograms
+The icon set of HDS only features user interface and social media icons. If you need illustrative icons or pictograms, please refer to the <Link size="M" href="https://brand.hel.fi/kuvitusohjeisto/" external>Helsinki Visual Identity Guidelines - Imagery guidelines (in Finnish)</Link>.
+
+### Size
+The default size of the HDS icons is 24×24px.
+- If needed, other sizes matching the [layout spacing tokens](/foundation/design-tokens/spacing "Spacing documentation") can also be used.
+- When using the icons in 16×16px, make sure the icon is still clear and legible.
+- Larger sizes than 64×64px are not recommended.
+
+<Image src="/images/foundation/visual-assets/icons/icon-sizes.png" alt="HDS icon sizes" style="width:100%; max-width: 550px;" viewable />
```
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply new-site/src/docs/foundation/visual-assets/icons/index.mdx:40]: Hmm, it does seem to work for me.

image

File: new-site/src/docs/components/linkbox/code.mdx [hunk 6]
```diff
@@ -142,15 +139,12 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
       href="https://hds.hel.fi"
       heading="Linkbox title"
       text="Linkbox text"
-      imgProps={{ src: "../../static/assets/placeholders/image/image-m@3x.png", width: 384, height: 245 }}
+      imgProps={{ src: "../../static/assets/placeholders/image/image-m@3x.png", maxWidth: 384, height: 245 }}
```
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply new-site/src/docs/components/linkbox/code.mdx]: - [ ] Fixed Linkbox with image examples' image path

File: new-site/src/components/layout.js [hunk 7]
```diff
@@ -177,7 +177,7 @@ const Layout = ({ children, pageContext }) => {
       .sort(sortByPageTitle),
   }));
   const footerCopyRightLinks = siteData?.footerCopyrightLinks || [];
-  const contentId = 'content';
+  const contentId = 'page-heading-1';
```
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply new-site/src/components/layout.js:180]: I do not fully understand this. What has to be changed?

File: new-site/src/docs/foundation/visual-assets/icons/index.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (minevala) [reply new-site/src/docs/foundation/visual-assets/icons/index.mdx:40]: I still get the: 
Screenshot 2022-06-03 at 15 24 56

Maybe it is not pushed to repository

File: new-site/src/components/layout.js (hunk 7 above)
[the remark this answers is not in the document]
Comment (minevala) [reply new-site/src/components/layout.js:180]: I'm not sure can we fix this in this way. Maybe we should use some other id 🤔

File: new-site/src/docs/foundation/visual-assets/icons/index.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply new-site/src/docs/foundation/visual-assets/icons/index.mdx:40]: - [ ] Added a missing icon sizes image

File: new-site/src/components/layout.js (hunk 7 above)
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply new-site/src/components/layout.js:180]: Ah, it is an ID and not a class. I will revert the change.

**Review by minevala — APPROVED** [review]
Nice fixes 🍰 
There is one conflict with versions in "what is new" but it should be manageable

**Comment by ronijaakkola** [comment]
Conflicts fixed. This should be ready to merge.

**Review by minevala — APPROVED** [review]
Nice fixes 👍

**Pull request merged**